package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "rdflib",
   "description": "an RDF library for node.js. Suitable for client and server side.",
-  "version": "0.21.1",
+  "version": "0.21.1-1",
```

Comment:
Unless you've published a prerelease to NPM, I don't think we should merge this.